docs/en/ingest-management/elastic-agent/configuring-kubernetes-metadata.asciidoc

```diff
@@ -0,0 +1,120 @@
+[[configuring-kubernetes-metadata]]
+= Configuring Kubernetes metadata enrichment on {agent}
+
+Kubernetes https://www.elastic.co/guide/en/observability/current/monitor-kubernetes.html#beats-metadata[metadata] refer to contextual information extracted from Kubernetes resources. Metadata information enrich metrics and logs
+collected from a Kubernetes cluster, enabling deeper insights in Kubernetes environments.
+
+When the elastic-agent's policy includes https://docs.elastic.co/en/integrations/kubernetes[{k8s} Integration] which configures the collection of Kubernetes related metrics and container logs, the mechanisms used for the metadata enrichment are:
+* <<kubernetes-provider>> for log collection
+* Kubernetes metadata enrichers for metrics
+
+In case the elastic-agent's policy does not include Kubernetes integration, but {agent} runs inside a Kubernetes
+environment the Kubernetes metadata are collected by the <<add_kubernetes_metadata-processor>>. The processor is configurable when {agent} is managed by {fleet}.
+
+
+[discrete]
+== Kubernetes Logs
+
+When it comes to container logs collection, the <<kubernetes-provider>> is used. It monitors for pod resources
+in the cluster and associates each container log file with a corresponding pod's container object.
+That way, when a log file is parsed and an event is ready to be published to ES, the internal mechanism knows to which actual
+container this log file belongs to. The linkage is established by the container's ID, which forms an integral part of the filename for the log.
+The Kubernetes autodiscover provider has already collected all the metadata for that container, leveraging pod, namespace and node watchers. Thus the events are enriched with the relevant metadata.
+
+In order to configure the metadata collection, the Kubernetes provider needs to be configured. 
+All the available configuration options of **Kubernetes provider** can be found in https://www.elastic.co/guide/en/fleet/current/kubernetes-provider.html[here].
+
+In more details:
```

Comment:
```suggestion

```

I suggest removing this line. I think people will understand the rest of the page to be examples.